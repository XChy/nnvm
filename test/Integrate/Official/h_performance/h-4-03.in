version https://git-lfs.github.com/spec/v1
oid sha256:181b2dc2aff4332823e76c06cb5534a3cece9471810e8f105724786db9b5b592
size 146
