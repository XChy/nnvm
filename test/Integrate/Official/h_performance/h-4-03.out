1
708208754
329960962
597932428
90988822
566711179
0